scenarios/reproducers/external-ceph.yml

```diff
@@ -0,0 +1,79 @@
+---
+cifmw_use_libvirt: true
```

Comment:
I think we can remove that `cifmw_use_libvirt: true` - it's already present in the "parent" parameter file.